### PR TITLE
Update Gateway & increase access token column length.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?
This pull request updates the Voting App to use [Gateway 1.3](https://github.com/DoSomething/gateway/releases/tag/v1.3.0), applies the included migration to increase the column length for access tokens to accommodate longer tokens, and switches to the `AES-256-CBC` cipher for compatibility with the latest Homestead.

#### How should this be reviewed?
Logging in via Northstar should continue to work.

#### Any background context you want to provide?
We're increasing the length of the private key used to sign tokens in Northstar to keep things nice and locked down, and that also increases the length of the generated tokens to over 1024 characters. The `AES-256-CBC` cipher (used for encrypting cookies & sessions) is removed in Laravel 5.3 and deprecated in PHP 7.1, so we're updating that to the new default.

#### What are the relevant tickets?
[Trello Card](https://trello.com/c/qLn2nRFR/590-2-as-a-developer-i-want-to-increase-the-key-length-for-northstar-tokens)

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

**Comment by DFurnes** [comment]
This is blocked on DoSomething/devops#241.

**Review by itsjoekent — APPROVED** [review]

**Comment by DFurnes** [comment]
Rebased after merging #553 and this is now looking good! 🏁

**Pull request merged**
